opentelemetry-sdk/src/opentelemetry/sdk/environment_variables/__init__.py

```diff
@@ -41,76 +41,119 @@
 OTEL_LOG_LEVEL = "OTEL_LOG_LEVEL"
 """
 .. envvar:: OTEL_LOG_LEVEL
+
+The :envvar:`OTEL_LOG_LEVEL` environment variable sets the log level used by the SDK logger
+Default: info
 """
 
 OTEL_TRACES_SAMPLER = "OTEL_TRACES_SAMPLER"
 """
 .. envvar:: OTEL_TRACES_SAMPLER
+
+The :envvar:`OTEL_TRACES_SAMPLER` environment variable sets the sampler to be used for traces.
+Sampling is a mechanism to control the noise introduced by OpenTelemetry by reducing the number
+of traces collected and sent to the backend
+Default: "parentbased_always_on"
 """
 
 OTEL_TRACES_SAMPLER_ARG = "OTEL_TRACES_SAMPLER_ARG"
 """
 .. envvar:: OTEL_TRACES_SAMPLER_ARG
+
+The :envvar:`OTEL_TRACES_SAMPLER_ARG` environment variable will only be used if OTEL_TRACES_SAMPLER is set.
+Each Sampler type defines its own expected input, if any.
+Invalid or unrecognized input is ignored,
+i.e. the SDK behaves as if OTEL_TRACES_SAMPLER_ARG is not set.
 """
 
 OTEL_BSP_SCHEDULE_DELAY = "OTEL_BSP_SCHEDULE_DELAY"
 """
 .. envvar:: OTEL_BSP_SCHEDULE_DELAY
+
+The :envvar:`OTEL_BSP_SCHEDULE_DELAY` represents the delay interval between two consecutive exports
 """
```

Comment:
```suggestion
"""Default: 5000
```

Reply:
`Default: 5000` should be inside the closing `"""`.